wporg-gp-translation-events.php

```diff
@@ -222,6 +227,18 @@ function submit_event_ajax() {
 		);
 		$response_message = 'Event updated successfully!';
 	}
+	// The user who creates the event will assist to it when it's published.
+	if ( ( 'create_event' === $action && 'publish' === $event_status ) || ( 'edit_event' === $action && 'publish' === $event_status && 'draft' === $previous_status ) ) {
```

Comment:
Would it be easier if we used the https://developer.wordpress.org/reference/hooks/transition_post_status/ hook?

Reply:
Done.